README.md

```diff
@@ -3,10 +3,52 @@
 Project containing the implementations of quantitative finance models.
 
 Models to be implemented:
-- Black-Scholes
-- CAPM
-- VaR (Value at Risk)
-- Modern Portfolio Theory
+1. Black-Scholes
+2. CAPM
+3. VaR (Value at Risk)
+4. Modern Portfolio Theory
 
-For the first three we plan to create two implementations, one classical 
+For the first three we plan to create two implementations, one classical
 and the other one using Monte-Carlo simulations.
+
+## Steps
+
+We need the following
+
+1. Setup a GitHub repository.
+2. A way of getting financial data in C++
+3. A library for Monte Carlo simulations in C++ (maybe we don't even
+   need the library; TBD, we could implement ourselves).
+4. A library for regressions, and basic matrix operations (potentially
+   from boost).
+5. Implement the actual models
+    1. Black Scholes
+    2. CAPM
+    3. VaR (Value at Risk)
+    4. Modern Portfolio Theory
+6.     Implement the Monte Carlo versions of the first three.
```

Comment:
Too much indentation causes the line to be turned into a comment.

Please leave only one space after "6.".

Reply:
Sure, nice catch — will do in rev 2.